Formula/crosstool-ng.rb

```diff
@@ -12,6 +13,7 @@ class CrosstoolNg < Formula
     sha256 "c119b94b8b4782935e7c2968d195cddcfd2faa089768c0a0bf84dfcb8d5713cc" => :yosemite
   end
 
+  depends_on "bash" => :build
```

Comment:
We definitely do not want that. Moreover, it ends up hardcoding the path to Homebrew bash in the installed version.